software/contrib/quantizer.py

```diff
@@ -39,21 +42,18 @@
 HALF_CHAR_WIDTH = int(CHAR_WIDTH / 2)
 
 
-class ScreensaverScreen:
+class ScreensaverScreen(Screensaver):
     """Blank the screen when idle
 
     Eventually it might be neat to have an animation, but that's
     not necessary for now
     """
     def __init__(self, quantizer):
+        Screensaver.__init__(self)
```

Comment:
Does MicroPython not implement `super`?

Reply:
Yes, it does. That's just me forgetting that keyword exists and falling back on Python 2.x habits. Fixed now.